### ISSUE TITLE
hyperparameter optimization

### ISSUE THREAD

**Issue by abhinaya-sinha** [abhinaya-sinha/VLQ-NN-Reweighting#1]
Learning rate and other hyperparameters need to be systemically optimized - right now we are just guessing.

**Comment by abhinaya-sinha** [comment]
Solved
